wandb/sdk/internal/internal_api.py

```diff
@@ -1029,10 +1029,19 @@ def get_project_run_queues(self, entity: str, project: str) -> List[Dict[str, st
 
         res = self.gql(query, variable_values)
         if res.get("project") is None:
-            raise Exception(
-                f"Error fetching run queues for {entity}/{project} "
-                "check that you have access to this entity and project"
-            )
+            # circular dependency: (LAUNCH_DEFAULT_PROJECT = model-registry)
+            if project == "model-registry":
+                msg = (
+                    f"Error fetching run queues for {entity} "
+                    "check that you have access to this entity and project"
+                )
+            else:
+                msg = (
+                    f"Error fetching run queues for {entity}/{project} "
+                    "check that you have access to this entity and project"
+                )
+
+            raise Exception(msg)
```

Comment:
why are you raising a general exception here?

Reply:
oh i see you kept the old functionality

Reply:
Yep, I can update it to be more specific if you want :)